source/plot/LinePlotData2D.ooc

```diff
@@ -12,30 +12,42 @@ LineStyle: enum {
 
 LinePlotData2D: class extends PlotData2D {
 	lineStyle: LineStyle { get set }
+	dataSeries: VectorList<FloatPoint2D> { get set }
```

Comment:
It's not temporary, and it's not for speed - it's because every function like `sum`, `sort`, `max`, `min` etc. needs to be implemented explicitly for each `T`. You can't add those functions to `VectorList`. So if you have a data series of `FloatPoint2D`, chances are you will have use of the functions defined in `FloatPoint2DVectorList`. If you don't, then fine, just don't go implementing your own `sum` etc. functions here (in case you had any intention of doing so).